SU-SWS/sulgryphon-nextjs
# SUL23-120 | @jdwjdwjdw | Structure and Style Places to Study Card

# READY FOR REVIEW

# Summary
- [SUL23-120](https://stanfordits.atlassian.net/browse/SUL23-120): FE-DEV | Style Places to Study card
<img width="861" alt="Screen Shot 2023-03-31 at 5 24 46 PM" src="https://user-images.githubusercontent.com/49299083/229235470-b70b203a-2aa9-4d19-a4b1-d6589ac726f0.png">
<img width="1164" alt="Screen Shot 2023-03-31 at 5 25 12 PM" src="https://user-images.githubusercontent.com/49299083/229235530-c094cf34-34bc-4242-b74b-eb3c5403c379.png">

# Review By (Date)
- When convenient

# Urgency
- How critical is this PR?

# Steps to Test

1. Checkout branch
2. Create a couple of branch locations
3. Edit the Places of Study Taxonomy so that terms are added
4. Create Places of Study based on those branch locations
5. Create basic page with Places of Study List
6. Review to make sure it matches designs
7. Review code

Three current known issues/questions:
- I updated the lists paragraph to be full width for Places of Study, but I am struggling at fixing all the other lists to not be full width (see https://deploy-preview-66--stanford-library.netlify.app/sws-test-329-example-lists)
- The Modal for places to study only shows the image every other time you click it, which appears to be related to the resize hook, as the Modal only seems to recognize `width` half the time. If needed, removing that hook altogether still results in an acceptable experience using breakpoints only.
- I'm not sure what is needed in the page-display file. For now I just have it saying `Place of Study`

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- [SUL23-120](https://stanfordits.atlassian.net/browse/SUL23-120): FE-DEV | Style Places to Study card

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)


[SUL23-120]: https://stanfordits.atlassian.net/browse/SUL23-120?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
